app/android/android-passport-reader/app/build.gradle

```diff
@@ -74,7 +74,7 @@ dependencies {
     implementation 'commons-codec:commons-codec:1.13'
 
     //Camera
-    implementation "com.github.RedApparat:Fotoapparat:2.7.0"
+    implementation "com.github.fotoapparat:fotoapparat:2.7.0"
```

Comment:
fix source so that fotoapparat can compile successfully.

build is currently broken on jitpack:
https://jitpack.io/#RedApparat/Fotoapparat

<img width="598" alt="Screenshot 2025-06-21 at 9 04 58 AM" src="https://github.com/user-attachments/assets/66b43e22-3ca2-4322-8aa5-27f161b835ac" />